### DISCUSSION TITLE
TIMEOUT

### DISCUSSION THREAD

**Discussion by YE-JR** [PaddlePaddle/PaddleSpatial#159] [BaiduKDDCup2022] [answered]
Dear Organizer: 
I showed a TIMEOUT error in a recent commit, but when I tested it locally, it took less than 10 minutes to test 134 turbines.
![21e0170af7fa991cb679197f75e1e93](https://user-images.githubusercontent.com/84713815/172598293-0a2b49eb-9745-44ef-b4db-a2e8feff8c66.png)
My team name is JRTeam.
Looking forward to your reply!

**Comment by ramber1836** [comment, marked as answer]
The time cost that your model makes one prediction, i.e. around 10 minutes, is too much. 
The TIMEOUT is configured as 10 hours for the online test, which means that the time spent should be no more than 185 seconds for each prediction on average (195 predictions in total).

**Comment by YE-JR** [comment]
Thank you for your reply!




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "PaddlePaddle/PaddleSpatial"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年6月8日(星期三) 晚上8:17
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [PaddlePaddle/PaddleSpatial] TIMEOUT (Discussion #159)





 
The time cost that your model makes one prediction, i.e. around 10 minutes, is too much.
 The TIMEOUT is configured as 10 hours for the online test, which means that the time spent should be no more than 185 seconds for each prediction on average (195 predictions in total).
